aip/general/0154.md

```diff
@@ -108,7 +108,7 @@ prefix as mandated by [RFC 7232][].
 - For how to retry on errors in client libraries, see AIP-194.
 
 ## Changelog
-
+- **2021-03-24**: Updated an additional reference to `FAILED_PRECONDITION` to `ABORTED`.
```

Comment:
- **2021-04-01**: Updated an additional reference to `FAILED_PRECONDITION`
  to `ABORTED`.